content/migrations/0035_relative_urls.py

```diff
@@ -0,0 +1,381 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+from django.db.models import Q
+import re
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName". 
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        regex = re.compile("http://.*/media/")
+        for item in orm.TestingQuestion.objects.filter(
+                Q(question_content__contains='/media/') | Q(answer_content__contains='/media/')):
+            item.question_content = regex.sub("/media/", item.question_content)
+            item.answer_content = regex.sub("/media/", item.answer_content)
+            item.save()
+
+    def backwards(self, orm):
+        "Write your backwards methods here."
```

Comment:
Should you not restore this to it's original state if we decide to roll it back?